### ISSUE TITLE
If pod log consists solely of a floating point number, read_namespaced_pod_log truncates log output

### ISSUE THREAD

**Issue by freedryk** [kubernetes-client/python#163]
I was playing with a batch job that simply outputs pi to 1000 decimal places.  When I look at the log output in the k8s dashboard it shows:

    3.1415926535897932384626433832795028841971693993751058209749...

but calling 

    core_api = client.CoreV1Api()
    core_api.read_namespaced_pod_log(name=pod_name, namespace=pod_namespace, container=container_name)

returns

    '3.14159265359'

I note that in Python, `str(3.14159265358979323846)` returns `3.14159265359`, so I'm guessing the client is converting the log into a float, then converting that float into a string.

**Comment by mbohlool** [comment]
That is an interesting case. I've traced it back to https://github.com/mbohlool/client-python/blob/4e593a7530a8751c817cceec715bfe1d03997793/kubernetes/client/api_client.py#L233-L233

I think we kind of hoped that json deserialization fails for logs and we use the response data directly. But in this case, it would deserialize output to json (that would be a floating point number) and then later on as we expect `str` we converted it to str. This needs more investigation.
